### DISCUSSION TITLE
RSS Feed 403 Forbidden

### DISCUSSION THREAD

**Discussion by paulkakell** [glanceapp/glance#234] [Help]
I'm having trouble with some rss feeds which open in a browser just fine, however when I'm trying to use them in the rss type in Glance it fails and in the container logs it is showing a 403 Forbidden error.  I'm assuming this is because of the headers Glance is providing the remote system.  Has anyone else run into this and is there a solution other than running my own rss aggregator to set the headers and then serving that directly to Glance?  

Thanks for any thoughts.

**Comment by svilenmarkov** [comment]
I've added a property that allows you to specify custom headers for each feed which will be available in v0.7.0.
